content/participate/report-issue.adoc

```diff
@@ -0,0 +1,202 @@
+---
+layout: project
+title: Issue Reporting
+section: participate
+tags:
+  - developer
+  - testing
+  - contributing
+---
+
+NOTE: This page is under development, there will be more content added soon.
+See the jira:WEBSITE-662[] EPIC for tasks related to this page, contributions are welcome!
+
```

Comment:
This page is complete with this change.  Nicely done!  We'll still accept pull requests to improve it, but the content is strong and worth describing as "done".
```suggestion
```